wrappers/workflow_wrapper.py

```diff
@@ -29,11 +31,12 @@ def create_pull_request_in_current_sprint(issue, jira, issue_text, assignee=None
     if isinstance(jira, MyJiraWrapper):
         if issue_text is not None:
             params = {
-                'summary': "{}#{} -{}".format(issue_text, assignee, issue.title),
-                'details': "{}".format(issue.data['body']),
+                'summary': "{} -{}".format(issue_text, issue.title),
+                'details': "Source : {}\n{}".format(issue.data['html_url'].encode(),
+                                                    issue.data['body']),
                 'component': 'Automation',
                 'labels': ['Automation'],
-                'sprint': jira.current_sprint_name.encode(),
+                'sprint': 'backlog',
```

Comment:
while testing I accidentally it was changed to the backlog. This will not work in local docker Jira instance, as we don't have sprint configured

Reply:
or we can build the sprint earlier in a try block:

```python
try:
    sprint =  jira.current_sprint_name.encode()
except WhateverExceptionitRaises:
   sprint = 'backlog'
```

then we use `'sprint': sprint`

Reply:
that's good, let me rephrase it.